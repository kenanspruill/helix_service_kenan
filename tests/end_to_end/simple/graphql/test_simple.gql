query getProviders {
    providers(
        query_id: "foo"
    ) {
        total_count
        results {
            result_id
        }
    }
}
