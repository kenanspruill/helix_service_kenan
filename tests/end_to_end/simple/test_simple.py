from pathlib import Path

from flask.testing import FlaskClient

from tests.end_to_end.test_runner import run_test_runner


def test_simple(graphql_client: FlaskClient) -> None:
    # Arrange
    data_dir: Path = Path(__file__).parent.joinpath("./")

    run_test_runner(
        data_dir=data_dir,
        graphql_client=graphql_client,
        test_name="test_simple",
    )
