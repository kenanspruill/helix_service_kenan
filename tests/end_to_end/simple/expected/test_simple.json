{
  "data": {
    "providers": {
      "results": [
        {
          "result_id": "123"
        }
      ],
      "total_count": 1
    }
  }
}
